nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -249,6 +249,7 @@ map_saver:
     save_map_timeout: 5000
     free_thresh_default: 0.25
     occupied_thresh_default: 0.65
+    map_subscribe_transient_local: True
```

Comment:
Should this be added to the other 2 yamls too?

Reply:
map_saver is not even in the other two yamls, should I add it?

Reply:
That's fine